docs/resources/servicestage_component_instance.md

```diff
@@ -150,7 +199,7 @@ The `refer_resource` block supports:
   + When `type` is set to **cce**, this parameter is mandatory, and need to specify the namespace of the cluster where
   the component is to be deployed, such as **{"namespace": "default"}**.
   + When `type` is set to **ecs**, this parameter is mandatory, and need to specify the hosts where the component is to
-  be deployed, such as **{"hosts":["04d9f887-9860-4029-91d1-7d3102903a69", "04d9f887-9860-4029-91d1-7d3102903a70"]}}**.
+  be deployed, such as **{"hosts":"[\"04d9f887-9860-4029-91d1-7d3102903a69\", \"04d9f887-9860-4029-91d1-7d3102903a70\"]"}**.
```

Comment:
the type of `refer_resource.paramater` is Map not String, please fix it.

Reply:
done.